src/components/UserMenu/UserMenuPresentation.js

```diff
@@ -105,33 +105,36 @@ function UserMenuPresentation({ currentUser, login, logout }) {
               <Menu.Item className="w-full flex">
                 {({ active }) => (
                   // eslint-disable-next-line jsx-a11y/anchor-is-valid, jsx-a11y/no-static-element-interactions
-                  <a type="button" onClick={login} onKeyDown={login}>
+                  <button type="button" onClick={login} onKeyDown={login}>
```

Comment:
If you are changing these to buttons we will no longer need the eslint-disable-next-line comments here and on line 122